### PR TITLE
9.12.7 RN

### PR DESCRIPTION
**Pull request by Adam-Dupaski**

### CODE REVIEW

File: content/en/docs/releasenotes/studio-pro/9/9.12.md [hunk 1]
```diff
@@ -3,13 +3,33 @@ title: "9.12"
...
+{{% modelerdownloadlink "9.12.7" %}}
+
+### Fixes
+
+* We fixed an issue where system members (such as `owner`, `changedBy`, `createdDate`, and `changedDate`) could not be selected on **Filter** and **Find** list operations in a microflow. (Ticket 151436)
+* We fixed an issue where uploaded images were not rotated in the upright position based on the orientation information in the image metadata. (Ticket 155278)
+* We fixed an issue where the periodic updating of sessions threw exceptions. (Ticket 162216)
+* We fixed an issue with the generate expression for changing an association on an object. (Ticket 163183)
+* We upgraded OpenJDK to 11.0.16.1+1 in the Studio Pro installer. (Ticket 164386)
+* We fixed an issue for workflows with **Decision** activities that caused the **Mark-as-Resolved** operation to hang.
```
Comment (arjenw) [inline content/en/docs/releasenotes/studio-pro/9/9.12.md]: This specific fix was last-minutely added to 9.12.6 actually. Can you remove it from 9.12.7 and add it to the 9.12.6 release notes?

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/studio-pro/9/9.12.md]: Can do, but which one - for 162216, 163183, 164386, or workflows? @arjenw

Comment (arjenw) [reply content/en/docs/releasenotes/studio-pro/9/9.12.md]: This one:
> * We fixed an issue for workflows with **Decision** activities that caused the **Mark-as-Resolved** operation to hang.

(for workflows)

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/studio-pro/9/9.12.md]: @MariaShaposhnikova for visibility on above for workflows, I'll make this move

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/studio-pro/9/9.12.md]: https://github.com/mendix/docs/commit/9724dce6034299f3f3e3f396ca5705e0b85f26a9

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/studio-pro/9/9.12.md]: https://github.com/mendix/docs/pull/5099/commits/dd9ddaebd9e55495734a0ae3007ae80a3807d292

**Pull request merged**